applications/GeoMechanicsApplication/custom_elements/axisymmetric_stress_state.cpp

```diff
@@ -65,4 +66,13 @@ Vector AxisymmetricStressState::CalculateGreenLagrangeStrain(const Matrix& rDefo
                     "configurations.\n";
 }
 
+const Vector& AxisymmetricStressState::GetVoigtVector()
```

Comment:
I would suggest to make this member `const`, i.e.:
```suggestion
const Vector& AxisymmetricStressState::GetVoigtVector() const
```

Reply:
added